### PR TITLE
fixed payment processing and payment pending pill alignment

### PR DESCRIPTION
**Pull request by sandy-fyle**
**fixed payment processing and payment pending pill alignment**
image
image

### CODE REVIEW

File: src/app/fyle/team-reports/team-report-card/team-report-card.component.scss [hunk 1]
```diff
@@ -83,6 +83,7 @@
     color: white;
     border-radius: 16px;
     padding: 6px 12px;
+    margin-left: -27px;
```
Comment (Dimple16) [inline src/app/fyle/team-reports/team-report-card/team-report-card.component.scss:86]: this will break in different resolutions

Comment (Dimple16) [reply src/app/fyle/team-reports/team-report-card/team-report-card.component.scss:86]: we shouldn't be chaning css, can you check how the grid is handled

**Review by Dimple16 — CHANGES_REQUESTED** [review]

File: src/app/fyle/team-reports/team-report-card/team-report-card.component.scss (hunk 1 above)
Comment (sandy-fyle) [reply src/app/fyle/team-reports/team-report-card/team-report-card.component.scss:86]: okay, looking into it.

Comment (sandy-fyle) [reply src/app/fyle/team-reports/team-report-card/team-report-card.component.scss:86]: will commit this once , re commit the changes in Team View Advance Revamp

**Review by Dimple16 — APPROVED** [review]

**Pull request merged**